SeaPublicWebsite.Data/Migrations/20241126150816_AddSolarPVAnswer.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace SeaPublicWebsite.Data.Migrations
+{
+    public partial class AddSolarPVAnswer : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<int>(
+                name: "SolarElectricPanels",
+                table: "PropertyData",
+                type: "integer",
```

Comment:
`integer` is a bit of a surprise? Why's that?

Reply:
It's the underlying type for the enum, the index of the value. 
From PropertyData.cs:
`public SolarElectricPanels? SolarElectricPanels { get; set; }`

Reply:
Oh right it's an enum, yes